### PR TITLE
Login styling

### PR DESCRIPTION
**Pull request by gestchild**
Applies styling from [zeplin designs](https://app.zeplin.io/project/5aab926b4b6efde01259e959/screen/60c0dda670910cb90cefbabd) and updates links

Screenshot 2021-06-11 at 17 04 42

### CODE REVIEW

File: common/views/components/SignIn/SignIn.tsx [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+import Space from '../styled/Space';
+import Layout12 from '../Layout12/Layout12';
+import styled from 'styled-components';
+
+const SignInContainer = styled.div`
+  display: flex;
+`;
+
+const StyledSignIn = styled(Space).attrs({
+  as: 'div',
```
Comment (davidpmccormick) [inline common/views/components/SignIn/SignIn.tsx:17]: I think `Space` is a `div` in the first place

File: common/views/components/SignIn/SignIn.tsx [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+const SignIn: FunctionComponent = () => {
+  const { user, isLoading } = useUserInfo();
+  const router = useRouter();
+  return router?.pathname !== '/item'
+    ? (!isLoading && (
+        <Layout12>
+          <SignInContainer>
+            <StyledSignIn
+              h={{
```
Comment (davidpmccormick) [inline common/views/components/SignIn/SignIn.tsx:32]: You could add the `v` and `h` attributes to the styled component declaration to tidy up the markup a bit?

**Review by davidpmccormick — APPROVED** [review]
Couple of nits

File: common/views/components/CataloguePageLayout/CataloguePageLayout.tsx [hunk 3]
```diff
@@ -1,7 +1,7 @@
 import { FunctionComponent, useEffect, useState } from 'react';
 import PageLayout, { Props as PageLayoutProps } from '../PageLayout/PageLayout';
 import InfoBanner from '../InfoBanner/InfoBanner';
-import SignIn from '@weco/catalogue/components/SignIn/SignIn';
+import SignIn from '../SignIn/SignIn';
 import { prefix } from '@weco/identity/src/utility/prefix';
```
Comment (jamesgorrie) [inline common/views/components/CataloguePageLayout/CataloguePageLayout.tsx:5]: Bikeshedding - are we sharing code between all apps now? Seems a whiff of coupling which is always a little dangerous/

File: common/views/components/PageLayoutDeprecated/PageLayoutDeprecated.js [hunk 4]
```diff
@@ -27,7 +27,7 @@ import Space from '../styled/Space';
 // $FlowFixMe (tsx)
 import GlobalInfoBarContext from '../GlobalInfoBarContext/GlobalInfoBarContext';
 // $FlowFixMe (tsx)
-import SignIn from '@weco/catalogue/components/SignIn/SignIn';
+import SignIn from '../SignIn/SignIn';
 // $FlowFixMe (tsx)
 import TogglesContext from '@weco/common/views/components/TogglesContext/TogglesContext';
```
Comment (jamesgorrie) [inline common/views/components/PageLayoutDeprecated/PageLayoutDeprecated.js:32]: Worth changing all the `@weco/common` bits while you're here?

File: common/views/components/SignIn/SignIn.tsx [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+  as: 'div',
+  className: font('hnm', 5),
+})`
+  margin-left: auto;
+  background: ${props => props.theme.color('smoke')};
+`;
+
+const SignIn: FunctionComponent = () => {
+  const { user, isLoading } = useUserInfo();
+  const router = useRouter();
```
Comment (jamesgorrie) [inline common/views/components/SignIn/SignIn.tsx:26]: We know this is clientside only?

File: common/views/components/CataloguePageLayout/CataloguePageLayout.tsx [hunk 6]
```diff
@@ -1,15 +1,17 @@
...
 
 type Props = {
   hideInfoBar?: boolean;
+  hideSignIn?: boolean;
```
Comment (jamesgorrie) [inline common/views/components/CataloguePageLayout/CataloguePageLayout.tsx:9]: Should these be `boolean` with defaults to `false`?

File: common/views/components/CataloguePageLayout/CataloguePageLayout.tsx [hunk 7]
```diff
@@ -1,15 +1,17 @@
...
 } & PageLayoutProps;
 
 const CataloguePageLayout: FunctionComponent<Props> = ({
   hideInfoBar,
+  hideSignIn,
```
Comment (jamesgorrie) [inline common/views/components/CataloguePageLayout/CataloguePageLayout.tsx:14]: If it worth making this 1 prop about `hideTopContent` so anything in this position isn't shown?

**Review by jamesgorrie — APPROVED** [review]

File: common/views/components/CataloguePageLayout/CataloguePageLayout.tsx (hunk 7 above)
Comment (gestchild) [reply common/views/components/CataloguePageLayout/CataloguePageLayout.tsx:14]: Good idea, I'll do that

**Pull request merged**